blockchain/thresholdstate.go

```diff
@@ -878,6 +878,56 @@ func (b *BlockChain) IsAutoRevocationsAgendaActive(prevHash *chainhash.Hash) (bo
 	return isActive, err
 }
 
+// isSubsidySplitAgendaActive returns whether or not the agenda to change the
+// block reward subsidy split to 10/80/10, as defined in DCP0010, has passed and
+// is now active from the point of view of the passed block node.
+//
+// It is important to note that, as the variable name indicates, this function
+// expects the block node prior to the block for which the deployment state is
+// desired.  In other words, the returned deployment state is for the block
+// AFTER the passed node.
+//
+// This function MUST be called with the chain state lock held (for writes).
+func (b *BlockChain) isSubsidySplitAgendaActive(prevNode *blockNode) (bool, error) {
+	const deploymentID = chaincfg.VoteIDChangeSubsidySplit
+	deploymentVer, ok := b.deploymentVers[deploymentID]
+	if !ok {
+		return true, nil
```

Comment:
This looks like boilerplate from the other methods, but I'm not clear what this implies.  Not found means it was not in `chaincfg.Params.Deployments`, which means:

1. "voting is not enabled for this network" as described in `isLNFeaturesAgendaActive`.  Like simnet where it is always active?
2. This deployment is ancient history, removed from `Params.Deployments`, and thus passed?  Presumably the plan is if an agenda fails then all the related code that would look for it would also be removed if it is also removed from `Params.Deployments`.

Reply:
Yes on #1.  For #2, in the case of failure, yes, that's correct.  It the case of success though, the idea is that the code would be changed so that the historical activation points for `mainnet`/`testnet` are added and then these methods would turn into "if that historical activation point is an ancestor of a given node, it's active, otherwise, it's not".